registry/storage/driver/swift/swift.go

```diff
@@ -0,0 +1,547 @@
+// Package swift provides a storagedriver.StorageDriver implementation to
+// store blobs in Openstack Swift object storage.
+//
+// This package leverages the ncw/swift client library for interfacing with
+// Swift.
+//
+// Because Swift is a key, value store the Stat call does not support last modification
```

Comment:
Could you include some documentation about how the mapping is done to the key-value storage? There are some details covered in delete that seem to require some extra knowledge around this aspect.